Documentation/bpf/architecture.rst

```diff
@@ -33,7 +33,7 @@ in the networking stack, the stack has not yet extracted metadata out of the
 packet. On the other hand, tc BPF programs are executed later in the kernel
 stack, so they have access to more metadata and core kernel functionality.
 Apart from tc and XDP programs, there are various other kernel subsystems as
-well which use BPF such as tracing (kprobes, uprobes, tracepoints, etc).
+well, which use BPF such as tracing (kprobes, uprobes, tracepoints, etc).
```

Comment:
We should as well fix the sentence here, “which use BPF such as tracing” is not very clear. And the “as well” in addition to “other” is a bit redundant. How about:

“Apart from tc and XDP programs, various other kernel subsystems use BPF, such as tracing (via kprobes, uprobes, tracepoints, for example).”